Maui.DataGrid/DataGrid.xaml.cs

```diff
@@ -50,6 +50,11 @@ public DataGrid()
         {
             _collectionView.ItemsSource = InternalItems;
         }
+
+        this.Loaded += (sender, e) =>
```

Comment:
```suggestion
        Loaded += (sender, e) =>
```
`this`is unnecessary here since the call will be made in ctor.